model/src/test/kotlin/utils/PurlUtilsTest.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.utils
+
+import io.kotest.assertions.assertSoftly
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+import java.io.File
+
+import org.ossreviewtoolkit.model.readValue
+
+class PurlUtilsTest : StringSpec({
+    "The purl test suite should pass" {
+        val testDataFile = File("src/test/assets/test-suite-data.json")
+        val testData = testDataFile.readValue<List<TestSuiteData>>()
```

Comment:
I believe it would be valuable to rename to `TestCase` and move the filtering up, as follows:
```
val testCases = File("test-suite-data.json").readValue<List<TestCase>>().filterNot { it.isInvalid } 

testCases.forAll { testCase ->
   ...
}
```

Reply:
I've introduced the `testCase(s)` variables, but kept `testData(File)`.